src/lib/shared/ui/optimal-image/image.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import {
+    parseTailwindSize,
+    serveAtPixelDensities,
+    getWeservUrl,
+  } from './tailwind-weserv-interop';
+
+  let _class = '';
+  export { _class as class };
+  export let src: string;
+  export let alt: string;
+
+  $: dimensions = parseTailwindSize(_class);
+</script>
+
+<picture class={_class}>
+  <source srcset={serveAtPixelDensities(src, dimensions, [1, 2], 'webp')} type="image/webp" />
+  <img
+    src={getWeservUrl(src, dimensions)}
+    srcset={serveAtPixelDensities(src, dimensions, [1, 2])}
+    {alt}
```

Comment:
You're already passing `{...$$restProps}`, why do you need to make the `alt` prop explicit?

Reply:
To make it required